cmd/tailscale/cli/cli_test.go

```diff
@@ -1568,3 +1569,15 @@ func TestDocs(t *testing.T) {
 	}
 	walk(t, root)
 }
+
+func TestDeps(t *testing.T) {
+	deptest.DepChecker{
+		GOOS:   "linux",
+		GOARCH: "arm64",
+		BadDeps: map[string]string{
+			"tailscale.com/feature/capture": "don't link capture code for its Lua",
```

Comment:
I could see putting the lua code in the CLI and _not_ `tailscaled` as reasonable too, but yeah it should be one or the other.

Reply:
Hm, good point. Maybe a new package just for the Lua is better.

Reply:
Moved the Lua to its own package that only the CLI uses. With new dep tests.